moped-editor/src/libs/GQLAbstract.js

```diff
@@ -448,6 +452,8 @@ class GQLAbstract {
         "}"
     );
 
+    // Previous comment indicated we don't support a second tier ordering directive,
+    // but the following code appears to support it.
```

Comment:
Here is my change. It's only a comment, and I would want to take it out of the PR prior to merging, but it's helpful in in the review process. 

Everything else in this file is linting.